roles/matrix-nginx-proxy/defaults/main.yml

```diff
@@ -557,6 +557,16 @@ matrix_nginx_proxy_synapse_media_repository_locations: []
 matrix_nginx_proxy_synapse_user_dir_locations: []
 matrix_nginx_proxy_synapse_frontend_proxy_locations: []
 
+# synapse content caching
+matrix_nginx_proxy_synapse_cache_enabled: false
+matrix_nginx_proxy_synapse_cache_path: "{{ matrix_nginx_proxy_data_path+'/cache' if matrix_nginx_proxy_synapse_cache_enabled else '/tmp/cache' }}"
```

Comment:
Did you mean `if matrix_nginx_proxy_enabled` instead of `if matrix_nginx_proxy_synapse_cache_enabled`?

Reply:
yeap you re right !

Reply:
and the conditional is the other way around, i.e.: `matrix_nginx_proxy_synapse_cache_path: "{{ '/tmp/cache' if matrix_nginx_proxy_enabled else matrix_nginx_proxy_data_path+'/cache' }}"`